build.gradle

```diff
@@ -3,7 +3,7 @@ task wrapper(type: Wrapper) {
 }
 
 allprojects {
-    version = "2.0.6"
+    version = "2.0.7"
```

Comment:
Could do. I wasn't sure because it has no outward changes for the client. That is unless clients are using something from `ExoPlayer`, which ideally they shouldn't do.